src/components/MovieDescription/MovieDescription.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Image, Space, Row, Col } from 'antd';
+
+export const MovieDescription = ({
```

Comment:
Please fix styles a little bit in this component
![image](https://user-images.githubusercontent.com/52933810/159816735-bfe1cc19-ec8f-49ee-a7b1-dbfa9fbbdbb8.png)